src/components/MenuItem.tsx

```diff
@@ -325,10 +330,10 @@ function MenuItem(
     const StyleUtils = useStyleUtils();
     const combinedStyle = [style, styles.popoverMenuItem];
     const {isSmallScreenWidth} = useWindowDimensions();
+    const animatedHighlightStyle = useAnimatedHighlightStyle({shouldHighlight: highlighted, height: 56});
```

Comment:
Since `MenuItem` is a generic component, I suggest passing `height` into `MenuItem` from within `WorkspaceInitialPage` as it is derived value from `styles.sectionMenuItem.height`. Further, I think it may be even better to send the `useAnimatedHighlightStyle` typed props directly from `WorkspaceInitialPage`.

Reply:
I'd agree that passing the height as a prop would be a good idea. However, I don't think we should move the animation logic out. The reason being, that we'll have to maintain the animation state for each menu item individually in the wrapper component if we go by this route.